aws_piper/aws_piper.py

```diff
@@ -29,10 +50,12 @@ def deploy(config_file='config.yaml'):
 
     pipeline_id = create_response.get('pipelineId')
     parameter_objects = translator.definition_to_api_parameters(read_json_file(cfg.get('parameter_objects')))
-    parameter_values = translator.definition_to_parameter_values(read_json_file(cfg.get('parameter_values')))
     pipeline_definition = translator.definition_to_api_objects(read_json_file(cfg.get('pipeline_definition')))
+    parameter_values = get_parameter_values(cfg)
+
+    send_query_to_S3(cfg)
```

Comment:
This is probably too specific. All pipeline models / definitions aren't going to require uploading data to s3. You can take a look here at those [examples](https://docs.aws.amazon.com/datapipeline/latest/DeveloperGuide/welcome.html) to have an idea of the range of use case data pipeline can cover.
Therefore, it should probably not be added to this global library.
It would be a better fit for our own private models repo, `aws-data-pipelines`.
Even there, it should only be conditional to models that require uploading data to s3.

Reply:
Ok, then I interpreted it differently. If that's the case - sure, I can try to move all the changes into aws-data-pipelines. I though that there we don't want to have any more python code :). If it's nor the case, no problem!